rllib/evaluation/collectors/simple_list_collector.py

```diff
@@ -191,7 +215,101 @@ def add_action_reward_next_obs(self, values: Dict[str, TensorType]) -> None:
                     sub_list.append(flattened[i])
         self.agent_steps += 1
 
-    def build(self, view_requirements: ViewRequirementsDict) -> SampleBatch:
+    def build_for_inference(self) -> SampleBatch:
+        """During inference we will a samplebatch with a batch size of 1.
+        This data will only include the data for the last recorded timestep.
```

Comment:
this comment may be improved?
it may not only include data from the last timestep though.
just say ```During inference, we will build a SampleBatch with a batch size of 1 that can then be used to run the forward pass of a policy```?

Reply:
done.